cmd/server/accounts/repository.go

```diff
@@ -26,22 +26,25 @@ type Repository interface {
 	updateAccountStatus(accountID string, status admin.AccountStatus) error
 
 	getEncryptedAccountNumber(customerID, accountID string) (string, error)
+
+	getLatestAccountOFACSearch(accountID string) (*ofacSearchResult, error)
+	SaveAccountOFACSearch(id string, result ofacSearchResult) error
 }
 
-func NewRepo(logger log.Logger, db *sql.DB) *sqlAccountRepository {
-	return &sqlAccountRepository{logger: logger, db: db}
+func NewRepo(logger log.Logger, db *sql.DB) *SqlAccountRepository {
+	return &SqlAccountRepository{logger: logger, db: db}
 }
 
-type sqlAccountRepository struct {
+type SqlAccountRepository struct {
```

Comment:
Was there a reason to export this `SqlAccountRepository`?

Reply:
No, reverting that.